src/components/UserAuth/Login.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import "../../styles/root.css";
+import {steve} from "../../assets/images";
```

Comment:
Suggestion- Try to use the absolute path as it increases the code readability.

Reply:
Okay I'll